game/ato/flightplans/waypointbuilder.py

```diff
@@ -494,59 +501,105 @@ def escort(
         )
 
     @staticmethod
-    def stopover(stopover: ControlPoint, name: str = "STOPOVER") -> FlightWaypoint:
-        """Creates a stopover waypoint.
+    def land_refuel(
+        control_point: ControlPoint, is_destination: bool = False
+    ) -> FlightWaypoint:
+        """Creates a land refuel waypoint.
 
         Args:
-            control_point: Pick up location.
+            control_point: The airbase to refuel.
+            is_destination: Due to an DCS Bug we need to set the last landing waypoint also as LandingReFuAr
```

Comment:
... but if it's the last waypoint we wouldn't use this method, we'd use `land`.

Reply:
see above